src/app/js/main.js

```diff
@@ -2,26 +2,29 @@ window.DATA = { 'log': [], 'responses': {} }
 
 const GALFILTERED = 'https://raw.githubusercontent.com/cytoscape/cytoscape-platform-tests-js/master/networks/galFiltered.cx'
 
-function toggleLog () {
+function toggleLog() {
   const log = document.getElementById('log-container')
   log.style.display = log.style.display === 'none' ? 'block' : 'none'
 }
 
-function init (slide) {
+function init(slide) {
   console.debug("Main init", slide, session);
-
-  addResponse(slide.id, { 'appVersion': window.navigator['appVersion'] })
+  // define test start time and add to response stack
+  var testDate = new Date()
+  addResponse(slide.id, { 'test_date': testDate })
```

Comment:
Does this need to be in the main.js file in the src directory? If it does, I can just move it to there.